link.html

```diff
@@ -0,0 +1,46 @@
+<html>
+<!--=================================================
+    STYLES AND PAGE INSTRUCTIONS. IGNORE THIS STUFF
+    =================================================-->
+<head>
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <style>
+    .outer { justify-self: center; margin: 20% auto 120px auto; width: 60%; ; font-family: 'Roboto', sans-serif;} a { text-decoration: underline; } li { padding-left: 1rem;}
+  </style>
+</head>
+<body style="background-color:#B1EEFC;">
+    <div class="outer">
+      This .html file is designed to be used with Plaid's <a href="https://github.com/plaid/plaid-postman">Postman Collection</a>. Please note the "Link Account" button below will not work until you've followed the steps in the <a href="https://github.com/plaid/plaid-postman">README</a>. </p>
```

Comment:
```suggestion
      This HTML file is designed to be used with Plaid's <a href="https://github.com/plaid/plaid-postman">Postman Collection</a>. Please note the "Link Account" button below will not work until you've followed the steps in the <a href="https://github.com/plaid/plaid-postman">README</a>. </p>
```

Reply:
I removed the closing paragraph tag, but ofc another option is to add in the corresponding opening tag.